lua/neotest-busted/init.lua

```diff
@@ -191,18 +194,9 @@ local function quote_string(string)
     return '"' .. string .. '"'
 end
 
----@param paths string[]
----@param options neotest-busted.TestCommandOptions?
----@return neotest-busted.TestCommandConfig?
----@diagnostic disable-next-line: inject-field
-function BustedNeotestAdapter.create_test_command(paths, options)
-    local busted_config = BustedNeotestAdapter.find_busted_command()
-
-    if not busted_config then
-        logging.error("Could not find a busted installation")
-        return
-    end
-
+---@param busted_config neotest-busted.BustedCommandConfig
+---@return arguments string[]
+local function get_nvim_arguments(busted_config)
```

Comment:
The sections adding `config.busted_paths`, `config.busted_cpaths`, `busted_config.lua_paths`, and `busted_config.lua_cpaths` have to be added in the general case, not just for neovim, as someone running `busted` directly might want to add custom paths and we still want to include the paths we find via `find_busted_command`.

```lua
    -- Add paths for busted
    vim.list_extend(lua_paths, busted_config.lua_paths)
    vim.list_extend(lua_cpaths, busted_config.lua_cpaths)
```

In relation to the above, please add tests in `adapter_build_spec_spec.lua` so we are testing that the right commands are being constructed and we don't break existing functionality.

Reply:
Makes sense, I updated the code to inject the paths via the environment in case `busted` is run without nvim.